packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -26,29 +39,51 @@ const Content = ({ icon, src, fill, content, stroke }) => {
  * It has two shapes: default and circle
  */
 const Avatar = forwardRef(
-  ({ icon = BuildingFilled, src, children, fill, stroke, ...props }, ref) => {
+  (
+    {
+      icon = BuildingFilled,
+      src,
+      children,
+      fill,
+      stroke,
+      borderRadius,
+      width,
+      height,
+      ...props
+    },
+    ref,
+  ) => {
     return (
-      <Box
-        ref={ref}
-        bgColor="elements.selectionAndIcons"
+      <Contain
+        width={width || 48}
+        height={height || 48}
+        borderRadius={borderRadius || 'small'}
```

Comment:
Maybe we can move the default value to the prop, just like we did to the `icon`.

```suggestion
        width={width}
        height={height}
        borderRadius={borderRadius}
```